packages/core/src/services/documentLogs/computeDocumentLogsWithMetadata.ts

```diff
@@ -12,55 +16,41 @@ export function getCommitFilter(draft?: Commit) {
     : isNotNull(commits.mergedAt)
 }
 
-export function getCommonQueryConditions({
-  scope,
-  documentUuid,
-  draft,
-  allowAnyDraft,
-}: {
-  scope: any
-  documentUuid?: string
-  allowAnyDraft?: boolean
-  draft?: Commit
-}) {
-  const byDocumentUuid = documentUuid
-    ? eq(scope.documentUuid, documentUuid)
-    : sql`1 = 1`
-
-  if (allowAnyDraft) return byDocumentUuid
-
-  return and(byDocumentUuid, getCommitFilter(draft))
+export function getDocumentLogFilters(options: DocumentLogFilterOptions) {
+  return and(
+    options.commitIds.length
+      ? or(inArray(documentLogs.commitId, options.commitIds))
```

Comment:
i think these ors are unnecessary

Reply:
Also I think it is good to make a set to discard any duplicated value:
`[...new Set(options.commitIds)]` & `[...new Set(options.logSources)]`